packages/app/src/LogSidePanel.tsx

```diff
@@ -2289,3 +2331,97 @@ export default function LogSidePanel({
     </Drawer>
   );
 }
+
+// TODO: move these to a separate file
+
+interface BreadcrumbTableItemProps {
+  breadcrumb: Breadcrumb;
+}
+
+/**
+ * Renders the type of a breadcrumb and its corresponding type
+ * @param breadcrumb
+ * @constructor
+ */
+function BreadcrumbType({ breadcrumb }: BreadcrumbTableItemProps) {
+  let icon = <i className="bi bi-info-circle" />;
+
+  switch (breadcrumb.category) {
+    case 'fetch': {
+      icon = <i className={'bi bi-arrow-left-right'} />;
+      break;
+    }
+    case 'ui.click': {
+      icon = <i className="bi bi-mouse" />;
+      break;
+    }
+    case 'navigation': {
+      icon = <i className="bi bi-geo-alt" />;
+      break;
+    }
+    case 'sentry.transaction': {
+      icon = <i className="bi bi-gear" />;
+    }
+  }
+
+  return <span className="me-2">{icon}</span>;
+}
+
+/**
+ * Renders the description of a breadcrumb
+ *
+ * Based on the type and category of the breadcrumb, displays different information
+ */
+function BreadcrumbDescription({ breadcrumb }: BreadcrumbTableItemProps) {
+  if (breadcrumb.type === 'http') {
+    const { method, status_code, url, ...rest } = breadcrumb.data ?? {};
+
+    return (
+      <div>
+        {method} {status_code} {url}
+        <pre className="p-2 bg-grey mt-1">
+          <code>{JSON.stringify(rest, null, 2)}</code>
+        </pre>
+      </div>
+    );
+  }
+
+  if (
+    breadcrumb.category === 'navigation' ||
+    breadcrumb.category === 'console'
```

Comment:
It looks like console breadcrumbs capture the message in `breadcrumb.message`, I think we might need some more robust logic here.